draft-ietf-quic-manageability.md

```diff
@@ -584,19 +598,55 @@ Handshake RTT can be measured by adding the client-to-observer and
 observer-to-server RTT components together. This measurement necessarily
 includes any transport and application layer delay at both sides.
 
-The spin bit
-provides an additional method to measure intraflow per-flow RTT.  When a QUIC
-flow is sending at full rate (i.e., neither application nor flow control
-limited), the latency spin bit on short header packets changes value once
-per round-trip time (RTT). An on-path observer can observe the time difference
-between edges in the spin bit signal in a single direction to measure one
-sample of end-to-end RTT. Note that this measurement, as with passive RTT
-measurement for TCP, includes any transport protocol delay (e.g., delayed
-sending of acknowledgements) and/or application layer delay (e.g., waiting for
-a request to complete). It therefore provides devices on path a good
-instantaneous estimate of the RTT as experienced by the application. A simple
-linear smoothing or moving minimum filter can be applied to the stream of RTT
-information to get a more stable estimate.
+# Using the Spin Bit for Passive RTT Measurement {#spin-usage}
+
+The spin bit provides an additional method to measure intraflow per-flow RTT
+from observation points on the network path throughout the duration of a
+connection. Support of the spin bit is an optional feature in QUIC. Usage of
+the spin bit mechanism must be enabled by both endpoints in order to function.
+Some endpoints may disable use of the the spin bit by default, others only in
+specific deployment scenarios, e.g. for servers and clients where the RTT would
+reveal the presence of a VPN or proxy. In order to not make these connections
+identifyable based on the usage of the spin bit, all endpoints are recommended
+to disable to spin randomly for at least one eighth of connections, even if
+otherwise enabled by default. If disable the connection would either use a
+fixed spin value for the while connection ("no spinning") or would set the
+spin bit randomly on each packet of the connection.
```

Comment:
This last sentence needs work.

Reply:
how about "An endpoint not participating in spin bit signaling for a given connection can use a fixed spin value for the duration of the connection, or can set the bit randomly on each packet sent."

Reply:
Sorry there were a bunch of nits in this sentence but used Brian's proposal now.